src/settings.js

```diff
@@ -1794,7 +1794,7 @@ var LEGACY_SETTINGS = [
   ['SKIP_STACK_IN_SMALL', [0, 1], 'SKIP_STACK_IN_SMALL is no longer needed as the backend can optimize it directly'],
   ['SAFE_STACK', [0], 'Replace SAFE_STACK=1 with STACK_OVERFLOW_CHECK=2'],
   ['MEMORY_GROWTH_STEP', 'MEMORY_GROWTH_LINEAR_STEP'],
-  ['WASM_OBJECT_FILES', [1], 'Use -flto or -fto=thin instead'],
+  ['WASM_OBJECT_FILES', [0, 1], 'Use -flto or -fto=thin instead'],
```

Comment:
typo in the flag, `fto` => `flto`

I'm not sure what is expected to happen here: this will accept the values 0,1 silently, which means the error will never be shown?

Separately, looks like we don't test `-flto=thin` yet - maybe worth adding to `other.test_lto_flags`?